src/main/java/mate/academy/service/impl/CinemaHallServiceImpl.java

```diff
@@ -0,0 +1,30 @@
+package mate.academy.service.impl;
+
+import java.util.List;
+import mate.academy.dao.CinemaHallDao;
+import mate.academy.lib.Inject;
+import mate.academy.lib.Service;
+import mate.academy.model.CinemaHall;
+import mate.academy.service.CinemaHallService;
+
+@Inject
+@Service
+public class CinemaHallServiceImpl implements CinemaHallService {
+    @Inject
+    private CinemaHallDao cinemaHallDao;
+
+    @Override
+    public CinemaHall add(CinemaHall cinemaHall) {
+        return cinemaHallDao.add(cinemaHall);
+    }
+
+    @Override
+    public CinemaHall get(Long id) {
+        return cinemaHallDao.get(id).get();
```

Comment:
Working with optional we shouldn't provoke NPE. This is our chance to use proper type of exception with meaningful message
```suggestion
        return cinemaHallDao.get(id).or.ElseThrow(() -> new EntityNotFoundException("some meaningful message"));
```